demo/molecule/buttonGroup/demo/index.scss

```diff
@@ -4,6 +4,14 @@
 
   &-section {
     margin: 10px;
+    
+    &--fullWidth {
+      border: 2px dashed blue;
+      overflow: hidden;
+      padding: 10px;
```

Comment:
No absolute values for marings, padding, etc...

Reply:
this is just some border style for the demo

Reply:
Sorry !!! 🤦‍♂️